bin/apisix

```diff
@@ -389,6 +389,10 @@ http {
         ssl_session_cache    shared:SSL:20m;
         ssl_session_timeout 10m;
 
+        {% if ssl.ssl_trusted_certificate ~= nil then %}
+        lua_ssl_trusted_certificate {* ssl.ssl_trusted_certificate *};
```

Comment:
Better to check if the certificate exists before generating the configuration.

Reply:
OK

Reply:
Added.